packages/controllers/src/snaps/SnapController.ts

```diff
@@ -722,6 +791,11 @@ export class SnapController extends BaseController<
       (...args) => this.has(...args),
     );
 
+    this.messagingSystem.registerActionHandler(
+      `${controllerName}:updateBlockedSnaps`,
```

Comment:
How/when do you imagine that this is called to check currently installed snaps?

Reply:
That's up to the consumer. In our case, we'll do it on booth, then either have a hardcoded list or poll an endpoint we set up.